ts/components/screens/BaseHeader.tsx

```diff
@@ -102,15 +103,19 @@ class BaseHeaderComponent extends React.PureComponent<Props, State> {
   public constructor(props: Props) {
     super(props);
     this.handleFocus = this.handleFocus.bind(this);
-    this.state = { isScreenReaderActive: false };
+    this.state = { isScreenReaderActive: false, isMounted: false };
   }
 
   // set accessibility focus when component is mounted
   // it should be used paired with avoidNavigationEvents === true (navigation context not available)
   public componentDidMount() {
+    this.setState({isMounted: true});
     void AccessibilityInfo.isScreenReaderEnabled()
```

Comment:
Not related to this PR, though I'm curious: why do we need a `void` in front of a function call?

Reply:
I think the reason is this [void operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void)

Reply:
Thanks @hevelius , though my question was _why_ rather than _what_ :-)

I see now that [@typescript-eslint/no-floating-promises](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-floating-promises.md) rule in the linter is the reason for us to use the operator.
And it makes sense since the function call is pure side-effect.

Today I learned something new... 🥳